plutus-pab/src/Plutus/PAB/Run/Command.hs

```diff
@@ -36,6 +37,19 @@ data ConfigCommand =
     deriving stock (Show, Eq, Generic)
     deriving anyclass JSON.ToJSON
 
+
+-- | A single command to the PAB that spins up all the necessary services.
+allServices :: ConfigCommand
```

Comment:
I did this just so I didn't have to duplicate it over the CliSpec and this test, and because maybe it's convenient to know.